packages/tables/resources/views/components/actions.blade.php

```diff
@@ -42,7 +42,9 @@ function ($action) use ($record): bool {
         ])
     }}
 >
-    @foreach ($actions as $action)
-        {{ $action }}
-    @endforeach
+    <div class="fi-ta-cell-content">
```

Comment:
Not sure about this.

Reply:
This adds the loading indicator to the actions cell